drivers/bridge/bridge.go

```diff
@@ -311,33 +312,22 @@ func (n *bridgeNetwork) isolateNetwork(others []*bridgeNetwork, enable bool) err
 		return nil
 	}
 
-	// Install the rules to isolate this networks against each of the other networks
-	for _, o := range others {
-		o.Lock()
-		otherConfig := o.config
-		o.Unlock()
-
-		if otherConfig.Internal {
-			continue
-		}
-
-		if thisConfig.BridgeName != otherConfig.BridgeName {
-			if err := setINC(thisConfig.BridgeName, otherConfig.BridgeName, enable); err != nil {
-				return err
-			}
-		}
+	// Install the rules to isolate this network against each of the other networks
+	if err := setINC(thisConfig.BridgeName, enable); err != nil {
+		return err
 	}
 
 	return nil
```

Comment:
Hrmm..  I didn't catch this the first time around, but when doing a quick build test locally, my machine flagged something that CI apparently didn't with golint.  Maybe I have a different golint version.
```
drivers/bridge/bridge.go:316:2: redundant if ...; err != nil check, just return error instead.
Makefile:109: recipe for target 'lint' failed
```
It is right:  it would be cleaner to change the last few lines to:
```go
return setINC(thisConfig.BridgeName, enable)
```
It builds fine once that change is made.  Sorry, for not catching this when requesting the error handling change.

Reply:
Agreed @ctelfer , If there is an error anyway we are not printing any error messages inside the if case.